src/DeviceInterface/AppleDeviceInterface.js

```diff
@@ -1,13 +1,73 @@
 const InterfacePrototype = require('./InterfacePrototype.js')
 const {wkSend, wkSendAndWait} = require('../appleDeviceUtils/appleDeviceUtils')
 const {
-    isApp, notifyWebApp,
+    isApp,
+    notifyWebApp,
+    isTopFrame,
     isDDGDomain,
     formatDuckAddress
 } = require('../autofill-utils')
 const {scanForInputs, forms} = require('../scanForInputs.js')
 
 class AppleDeviceInterface extends InterfacePrototype {
+    constructor () {
+        super()
+        if (isTopFrame) {
+            this.stripCredentials = false
+            window.addEventListener('mouseMove', this)
+        } else {
+            // This is always added as a child frame needs to be informed of a parent frame scroll
+            window.addEventListener('scroll', this)
```

Comment:
`removeEventListener("scroll", this.handleEvent)` also is wrong because the DOM binds 'this' to something else (and it's not consistent either between which listeners do and don't do this :( )